### ISSUE TITLE
Missing information on how to opt-out from SSDP

### ISSUE THREAD

**Issue by Unambiguous** [home-assistant/home-assistant.github.io#29683]
### Feedback

The page [Synology DSM - Home Assistant](https://www.home-assistant.io/integrations/synology_dsm/) mentions that SSDP may be causing an issue when more NICs are configured on the device. It suggests that deactivating SSDP may resolve the issue. 

I was trying to set the Synology DSM integration to opt-out of using SSDP, but based on the available documentation I could not do it. I suppose I should edit `manifest.json` inside synology_dsm integration folder, but I can not find such folder on my installation.

I would suggest the following changes to the documentation:
1. The mentioned page could add a hyperlink on SSDP inside the note that would point to https://www.home-assistant.io/integrations/ssdp/ .
2. The [SSDP documentation page](https://www.home-assistant.io/integrations/ssdp/) could add information with an exampole on how an integration can opt-out from using SSDP.

### URL

https://www.home-assistant.io/integrations/ssdp/

### Version

2023.11.0

**Comment by mib1185** [comment]
the docs says, you should deactivate SSDP on the NAS, not HA 😉

**Comment by Unambiguous** [comment]
In the meanwhile I found out that it was meant to be deactivated on the NAS itself. I found a page related to this setting on the Synology site - [Advanced | DSM - Synology Knowledge Center](https://kb.synology.com/en-id/DSM/help/DSM/AdminCenter/file_service_advanced_introduction?version=7).

I would suggest adjusting the information in the existing note to precisely define its context, i.e. the settings should be applied on the NAS. It may help other users that are not so familiar with SSDP. Something like:

> ... In this case, it is recommended use the following settings on the NAS itself - either use NIC bonding or deactivate SSDP.

Thank you.
